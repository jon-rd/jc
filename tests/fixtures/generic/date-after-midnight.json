{"year": 2021, "month": "Mar", "month_num": 3, "day": 24, "weekday": "Wed", "weekday_num": 3, "hour": 12, "hour_24": 0, "minute": 54, "second": 47, "period": "AM", "timezone": "UTC", "utc_offset": "+0000", "day_of_year": 83, "week_of_year": 12, "iso": "2021-03-24T00:54:47+00:00", "epoch": 1616572487, "epoch_utc": 1616547287, "timezone_aware": true}
