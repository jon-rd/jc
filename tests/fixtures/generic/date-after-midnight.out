Wed Mar 24 12:54:47 AM  UTC 2021
